lib/experimental/fonts/font-library/class-wp-font-library.php

```diff
@@ -27,6 +27,63 @@ class WP_Font_Library {
 		'woff2' => 'font/woff2',
 	);
 
+	/**
+	 * Font collections.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @var array
+	 */
+	public static $collections = array();
+
+	/**
+	 * Register filter to extend the library with font collections.
+	 *
+	 * @since 6.4.0
+	 */
+	public static function register_filters() {
```

Comment:
Just use $this here.  No need for this to a static method

Reply:
@matiasbenedetto this method is not needed. Instead, move its code to `font-library.php` file.  Rather than invoking this method which then hooks into the filter, directly hook the filter without the method.

Less code to maintain.

Reply:
>  move its code to font-library.php file. Rather than invoking this method which then hooks into the filter, directly hook the filter without the method.

Nice idea. Done!